### PR TITLE
Adding examples for Redis Cache top-level resource.

### PR DESCRIPTION
**Pull request by TimLovellSmith**
@amarzavery
Here are examples following proposed example format. Please review to see if there are any issues. 

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

**Comment by TimLovellSmith** [comment]
@amarzavery Note, I get validation errors from the Linter extension for VS Code for this PR because it doesn't like "x-ms-examples" - with the linter versoin I am using, anyway.

File: arm-redis/2016-04-01/examples/RedisCacheImport.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "api-version": "2016-04-01",
+    "subscriptionId": "subid",
+    "parameters": {
+      "format": "RDB",
+      "files": [ "http://fileuris.contoso.com/pathtofile1" ]
+    }
+  },
+  "responses":{
+    "202":{
+      headers": {
```
Comment (amarzavery) [inline arm-redis/2016-04-01/examples/RedisCacheImport.json]: missing starting double quotes.

File: arm-redis/2016-04-01/swagger/redis.json [hunk 2]
```diff
@@ -417,6 +441,9 @@
           "Redis"
         ],
         "operationId": "Redis_ImportData",
+        "x-ms-examples": { 
+          "RedisCacheImport": { "$ref": "../examples/RedisCacheImport.json" } 
+        },
         "x-ms-long-running-operation": true,
         "description": "Import data into Redis cache.",
         "parameters": [
```
Comment (amarzavery) [inline arm-redis/2016-04-01/swagger/redis.json]: 202 has an empty description.

**Comment by amarzavery** [comment]
@TimLovellSmith 
I had to add a double quote`"` in the "headers" key of the "202" response for the example of RedisCacheImport.json. There was an extra curly brace `}` in the end, which had to be removed. After that I was able to run the validation tool and found following issues:

#### The following operations do not have examples provided for them:
- Redis_Update
- PatchSchedules_CreateOrUpdate
- PatchSchedules_Delete
- PatchSchedules_Get

#### Body Parameter for Redis_Create did not have location, whereas the swagger spec says that location is a required property.
> Operation: Redis_Create

		> x-ms-examples

			> Scenario: RedisCacheCreate
				> BODY_PARAMETER_VALIDATION_ERROR: Found errors in validating the body parameter for example "RedisCacheCreate" in operation "Redis_Create".
				> InnerErrors:
				  1. code: OBJECT_MISSING_REQUIRED_PROPERTY
				     message: Missing required property: location
				     path: 
				     description: The Resource definition.

#### Please provide examples for all the responses for a given operation.
/cc @johanste @veronicagg @salameer

**Comment by TimLovellSmith** [comment]
@amarzavery I've pushed a new commit that I think addresses all the issues except the long running operation description.

**Review by amarzavery — APPROVED** [review]

**Pull request merged**